### ISSUE TITLE
Add an option to initialize pycondor jobs with arguments.

### ISSUE THREAD

**Issue by zdgriffith** [jrbourbeau/pycondor#89]
#### Description

Currently arguments must be added to a Job via `add_args()`.  Enabling a single set of arguments to be passed during instantiation would provide an easier way to mimic the original dagman workflow.

**Comment by jrbourbeau** [comment]
Good idea @zdgriffith, I totally agree! I think for cases where only a single argument is being added to a `Job` object that 

```python
job = Job('myjob', ex=ex, argument='some command line argument')
```

is much cleaner that 

```python
job = Job('myjob', ex=ex)
job.add_arg('some command line argument')
```

Do you have an interest in submitting a PR?

**Comment by zdgriffith** [comment]
Sure, I can take it.

**Comment by jrbourbeau** [comment]
Closed in  #90
